spec/requests/throttling_spec.rb

```diff
@@ -0,0 +1,33 @@
+RSpec.describe "Throttling" do
+  before do
+    Rack::Attack.enabled = true
+    Rack::Attack.cache.store = ActiveSupport::Cache::MemoryStore.new
+  end
+
+  after do
+    Rack::Attack.enabled = false
+  end
+
+  context "POST /" do
+    it "does not throttle" do
+      32.times { post "/" }
+      expect(response.body).to_not have_content(I18n.t("standard_errors.too_many_requests.heading"))
+    end
+
+    context "with user[email] set" do
+      it "throttles" do
+        32.times { post "/", params: { "user[email]" => "email@example.com" } }
+        follow_redirect!
+        expect(response.body).to have_content(I18n.t("standard_errors.too_many_requests.heading"))
+      end
+    end
+  end
+
+  context "POST /login" do
+    it "throttles" do
+      32.times { post "/login" }
```

Comment:
Could i just check this, the commit says 16 times but we're testing 32 times here?
Why did you go with double? Is there something that halves the requests, or is it just a big number over the limit?

If it's the over the limit thing I wonder if we could create a constant like:
`THROTTLE_REQUEST_LIMIT = 16.freeze`

Then do `(THROTTLE_REQUEST_LIMIT + 1).times` here?

Reply:
Aha, I can see how that could be confusing, I just doubled it for good measure.  I'll change this.